app/frontends/workflow-ui.ts

```diff
@@ -270,7 +270,8 @@ function workItemRenderingFunctions(job: Job, isAdmin: boolean, requestUser: str
     workflowItemUpdatedAt(): string { return this.updatedAt.getTime(); },
     workflowItemLogsButton(): string {
       const isComplete = [WorkItemStatus.FAILED, WorkItemStatus.SUCCESSFUL].indexOf(this.status) > -1;
-      if (!isComplete || !isAdmin || this.serviceID.includes('query-cmr')) return '';
```

Comment:
I prefer to prefix booleans with `is` so `isLogsAvailalbe` or maybe `areLogsAvailable` if that sound too crazy.